doc/docs/Mode_Decomposition.md

```diff
@@ -217,9 +229,19 @@ The Poynting flux carried by the Meep fields may be expressed in the form:
 $$ S_x = \frac{1}{2}\text{Re }
          \left\langle \mathbf{E}^{\text{meep}}\right|
          \left.       \mathbf{H}^{\text{meep}}\right\rangle
-       = \frac{1}{2}\sum_n \left\{ |\alpha_n^+|^2 - |\alpha_n^-|^2) \right\} v_n A_S
+       = \frac{1}{2}\sum_n \left\{ |a_n^+|^2 - |a_n^-|^2) \right\} v_n A_S
 $$
 
-Thus, the fractional power carried by a given forward- or backward-traveling eigenmode is given by:
+Thus, the power carried by a given forward- or backward-traveling eigenmode is given by:
+
+$$ \textit{power} = \frac{|a_n^\pm|^2 v_n A_S}{2S_x} $$
+
+or
+
+$$ \textit{power} = |\alpha_n^\pm|^2
+
+where we have defined $\alpha_n^\pm \equiv \left(\sqrt{v_n A_S}{2S_x}\right)a_n^\pm$.
+These $\{\alpha_n^\pm\}$ coefficients are the quantities reported by
+`get_eigenmode_coefficients.`
+
 
-$$ \frac{|\alpha_n^\pm|^2 v_n A_S}{2S_x} $$     
```

Comment:
@oskooi, the text was changed to say "power" instead of "fractional power", but it looks like we forgot to remove the S_x in the denominator, so the formula is now incorrect.